build_docs.sh

```diff
@@ -1,11 +1,21 @@
 #!/bin/bash
+# To build manually run "sudo sh build_docs.sh"
 gem install jazzy
 echo -e "Copying MSAL public files"
-mkdir docs.temp
-mkdir docs.temp/MSAL
+mkdir -m 777 docs.temp
+mkdir -m 777 docs.temp/MSAL
 cp `find MSAL/src/public` docs.temp/MSAL
+cp `find MSAL/src/native_auth/public` docs.temp/MSAL
 cp README.md docs.temp/
-cd docs.temp
+
 echo -e "Generating MSAL documentation"
-jazzy --objc --umbrella-header MSAL/MSAL.h --framework-root . --sdk iphonesimulator --author Microsoft\ Corporation --author_url https://aka.ms/azuread  --github_url https://github.com/AzureAD/microsoft-authentication-library-for-objc --theme fullwidth
+# Generate Swift SourceKitten output
+sourcekitten doc -- -workspace MSAL.xcworkspace -scheme "MSAL (iOS Framework)" -configuration Debug RUN_CLANG_STATIC_ANALYZER=NO -sdk iphonesimulator CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -destination 'platform=iOS Simulator,name=iPhone 14,OS=16.4' > docs.temp/swiftDoc.json
```

Comment:
Would there be a way to avoid hardcoding the exact version and device details in the destination here but instead use latest or something? We don't want this to break in the future when minimum versions, etc. change.

Reply:
if the name is supplied it has to be exact. But it's actually not required so I removed it. By default the first simulator installed will be used and since it's only building to create the documentation that will pass regardless of which it is.